Dispatch/DBhelper.py

```diff
@@ -1,89 +1,123 @@
 from Dispatch import config
 import psycopg2
 
-def build_connection():
-    connection = psycopg2.connect(database=config.database_name,
-                                  user=config.user_name,
-                                  password=config.password,
-                                  port=config.port,
-                                  )
-    return connection
+class Helper:
+    def __init__(self):
+        self.connection = psycopg2.connect(database=config.database_name,
+                                      user=config.user_name,
+                                      password=config.password,
+                                      port=config.port,
+                                      )
+        self.cursor = self.connection.cursor()
 
-def insert_trip(trip_id,user_id,pickup_location,dropoff_location):
-    pickup_location = "{"+f"{pickup_location[0]}"+","+f"{pickup_location[1]}"+"}"
-    dropoff_location = "{"+f"{dropoff_location[0]}"+","+f"{dropoff_location[1]}"+"}"
-    cursor = connection.cursor()
-    query = "INSERT INTO trip_table " \
-            "(trip_id,user_id,status,pickup_location,dropoff_location)" \
-            f"VALUES ('{trip_id}',{user_id},'created',"+"'"+f"{pickup_location}"+"'"+","+"'"+f"{dropoff_location}"+"'"+");"
-    cursor.execute(query)
-    connection.commit()
+    def insert_trip(self,trip_id,user_id,pickup_location,dropoff_location):
+        pickup_location = "{" + f"{pickup_location[0]}" + "," + f"{pickup_location[1]}" + "}"
+        dropoff_location = "{" + f"{dropoff_location[0]}" + "," + f"{dropoff_location[1]}" + "}"
+        query = "INSERT INTO trip_table " \
+                "(trip_id,user_id,status,pickup_location,dropoff_location)" \
+                f"VALUES ('{trip_id}',{user_id},'created'," + "'" + f"{pickup_location}" + "'" + "," + "'" + f"{dropoff_location}" + "'" + ");"
+        self.cursor.execute(query)
+        self.connection.commit()
 
-def get_trip_detail(trip_id):
-    cursor = connection.cursor()
-    query = "SELECT pickup_location,dropoff_location FROM trip_table " \
-            f"WHERE trip_id = '{trip_id}'"
-    cursor.execute(query)
-    connection.commit()
-    res = cursor.fetchall()
-    pickup_location = res[0][0]
-    dropoff_location = res[0][1]
-    return trip_id, pickup_location, dropoff_location
+    def get_trip_detail(self,trip_id):
+        query = "SELECT pickup_location,dropoff_location FROM trip_table " \
+                f"WHERE trip_id = '{trip_id}'"
+        self.cursor.execute(query)
+        self.connection.commit()
+        res = self.cursor.fetchall()
```

Comment:
If only choose 1 you can use fetchMany(size=1)

Reply:
https://pynative.com/python-cursor-fetchall-fetchmany-fetchone-to-read-rows-from-table/